app/api/search/specs/entitiesIndex.spec.js

```diff
@@ -131,9 +131,16 @@ describe('entitiesIndex', () => {
       let response = await checkMapping(templateB, elasticIndex);
       expect(response).toEqual({ errors: [], valid: true });
 
-      templateB.properties[0].type = 'text';
+      templateB.properties = [
+        { name: 'dob', type: 'text', label: 'Date of birth' },
+        { name: 'country', type: 'select', label: 'Country' },
+      ];
+
       response = await checkMapping(templateB, elasticIndex);
-      expect(response).toEqual({ errors: [{ name: 'Date of birth' }], valid: false });
+      expect(response).toEqual({
+        errors: [{ name: 'Date of birth' }, { name: 'Country' }],
+        valid: false,
+      });
```

Comment:
I'm not sure what test case covers this test. I commented the added condition in equalPropMapping and all the tests still passed. Please could you add a test to cover the specific scenario?

Reply:
So to the "sameProps" condition I added two conditions, the line  186 and the line 190, if you coment both it actually throws an error because of the property select that was added to the test.